core/feature_flag_list.py

```diff
@@ -112,7 +111,6 @@ class FeatureNames(enum.Enum):
     FeatureNames.ENABLE_VOICEOVER_CONTRIBUTION,
     FeatureNames.AUTO_UPDATE_EXP_VOICE_ARTIST_LINK,
     FeatureNames.ADD_VOICEOVER_WITH_ACCENT,
-    FeatureNames.DIAGNOSTIC_TEST,
```

Comment:
Note that this should move to the "deprecated" section.

Reply:
Done.